packages/components/inputs/async-select-input/src/async-select-input.tsx

```diff
@@ -193,6 +193,12 @@ type TAsyncSelectInputProps = {
    * [Props from React select was used](https://react-select.com/props)
    */
   menuShouldBlockScroll?: ReactSelectAsyncProps['menuShouldBlockScroll'];
+  /**
+   * When set to false, the select menu will not be closed after an option is selected. By default, it is set to true.
+   * <br>
+   * [Props from React select was used](https://react-select.com/props)
+   */
+  closeMenuOnSelect: ReactSelectProps['closeMenuOnSelect'];
```

Comment:
I don't think this should be required. 
```suggestion
  closeMenuOnSelect?: ReactSelectProps['closeMenuOnSelect'];
```